FrontEndApp/v1-react/src/resources/keras/kLayers.js

```diff
@@ -0,0 +1,557 @@
+// temp={
+//   Conv2D:{
+//     parameters:{
+//       filters:{
+
+//       }
+//     },
+//     dimensions:{
+//          expected_dim: 3,
+//          returned_dim: 3,
+//     },
+//     config:{
+
+//     }
+//   }
+// }
+
```

Comment:
Do we actually need this?
If yes, then can we have it more generic:
```
Layer: {
    parameters: {
    }
    ...
}
```

Reply:
more generic in the sense ?